cli/src/main/java/com/box/l10n/mojito/cli/command/PseudoLocCommand.java

```diff
@@ -85,29 +87,28 @@ public void execute() throws CommandException {
 
         for (FileMatch sourceFileMatch : commandHelper.getSourceFileMatches(commandDirectories, fileType, sourceLocale, sourcePathFilterRegex)) {
             consoleWriter.a("Localizing: ").fg(Color.CYAN).a(sourceFileMatch.getSourcePath()).println();
-            generateLocalizedFiles(repository, sourceFileMatch);
+            generatePseudoLocalizedFiles(repository, sourceFileMatch);
         }
         consoleWriter.fg(Color.GREEN).newLine().a("Finished").println(2);
     }
 
     /**
      * Default generation, uses the locales defined in the repository to
```

Comment:
This doc is irrelevant here, that was for the pull command that used to iterate on locales. Just say that it generates the pseudo localized file